charter/index.html

```diff
@@ -238,23 +244,20 @@ <h3>
-                        </p>
-                        <p class="draft-status">
-                          <b>Draft state:</b> Adopted from <a href="https://www.w3.org/groups/cg/solid">W3C Solid Community Group</a>.
+                            The PUMPKIN Protocol specification aims to provide applications with secure and permissioned access to externally stored data in an interoperable way.
                         </p>
-
                         <p>
-                          <b>Adopted Draft</b>: <a href="https://solidproject.org/ED/protocol">Solid Protocol, latest published version (0.10.0) or Editor's Draft</a>
+                            The PUMPKIN Protocol may include protocol details for integration with identity layers and mechanisms; access management and data integrity; notifications about resource changes; and, authorization mechanisms.
                         </p>
                         <p>
-                            When possible, the Solid Protocol will evolve while maintaining a high degree of compatibility with existing implementations, of both servers and clients, and with features from prior versions. If incompatible changes have to be made, then they will be done by introducing a stage where both old and new protocols are supported, to allow the implementors to upgrade their systems in a managed way.
+                          <b>Input document</b>: <a href="https://solidproject.org/ED/protocol">Solid Protocol, latest published version (0.10.0) or Editor's Draft</a>, adopted from the <a href="https://www.w3.org/groups/cg/solid">W3C Solid Community Group</a>.
+                             <q lang="en-GB" cite="https://solidproject.org/TR/2022/protocol-20221231#introduction">An overarching design goal of the Solid ecosystem is to be evolvable and to provide fundamental affordances for decentralized web applications for information exchange in a way that is secure and privacy respecting. In this environment, actors allocate identifiers for their content, shape and store data where they have access to, set access controls, and use preferred applications and services to achieve them.</q>
                         </p>
-                        <p>
-                            The Solid Protocol may include protocol details for integration with identity layers and mechanisms; access management and data integrity; notifications about resource changes; and, authorization mechanisms.
+                        <p class=new>
+                          The Working Group may consider other input, in addition to this specification, that is relevant to its scope and to the goal of this deliverable.
```

Comment:
```suggestion
```

I think this paragraph is unnecessary (as hinted by Ted). Furthermore, the details on other input/proposals is already explained later in the text:

>The Working Group will adopt proposals that fulfill the requirements of <a href="https://www.w3.org/Guide/standards-track/#criteria">W3C Recommendation Track Readiness</a> [..]

Reply:
In an ideal world, I agree it would be unnecessary. Given the concerns about rubber-stamping that have been expressed about the previous charter, I'm leaning towards keeping it. *Bis repetita placent*

Reply:
@csarven 
> I think this paragraph is unnecessary (as hinted by Ted).

If I hinted at that, it was accidental, and should be disregarded. (I can find no such hint within this PR... Can you link me to it?)

Reply:
@pchampin Okie dokie!

@TallTed Perhaps I've misinterpreted you in https://github.com/solid/solid-wg-charter/pull/69/files#r1521554997 but I seem to share the same interpretation as @pchampin re uncountable. But you're right, you didn't hint "unnecessary" and I was jumping ahead with "uncountable" input, which I think is already covered by later text.

Any way... is it dinner time?